### PR TITLE
Test GraphQL CLI for all categories

### PR DESCRIPTION
**Pull request by chrzaszcz**
Finish testing GraphQL with CLI in a unified way, making use of generated queries.

Categories included in this PR:
- [x] domain
- [x] http_upload
- [x] inbox
- [x] last
- [x] muc
- [x] muc_light
- [x] offline
- [x] private
- [x] roster
- [x] session
- [x] stanza

Also: fix a race condition in `offline_SUITE`, that for some reason started presenting itself in this PR.

### CODE REVIEW

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/129873) / small_tests / 137400b26a1997e12d52bb474476837836c03a1c
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3710/129873/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3710/129873/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2022-07-15_07.55.53/lib.mongooseim.logs/run.2022-07-15_07.55.54/suite.log.html)

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/129955) / small_tests / 93388ffb06b560393d8fc9ca0e264d534f04dedf
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3710/129955/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3710/129955/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2022-07-15_11.06.19/lib.mongooseim.logs/run.2022-07-15_11.06.20/suite.log.html)

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/130031) / small_tests / 3e36a7af4878c20ac6183f81663566880670a1d6
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3710/130031/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3710/130031/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2022-07-15_13.08.53/lib.mongooseim.logs/run.2022-07-15_13.08.53/suite.log.html)

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/130084) / small_tests / e2d3a9503c546c5f7e7ff3b7527e27b276c09dca
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3710/130084/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3710/130084/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2022-07-15_13.59.09/lib.mongooseim.logs/run.2022-07-15_13.59.09/suite.log.html)

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/130107) / small_tests / a3420a7933db35fa6953e75cbea3d5b58bb5f53e
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3710/130107/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3710/130107/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2022-07-18_07.57.11/lib.mongooseim.logs/run.2022-07-18_07.57.11/suite.log.html)

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/130135) / small_tests / e4d2fbce57fe830c3547d61eed27bc6ed2058ca0
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3710/130135/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3710/130135/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2022-07-18_11.57.21/lib.mongooseim.logs/run.2022-07-18_11.57.22/suite.log.html)

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/130160) / small_tests / a7a8645e266dc0030c3ad8bcb365ec2408d4286d
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3710/130160/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3710/130160/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2022-07-18_12.49.05/lib.mongooseim.logs/run.2022-07-18_12.49.05/suite.log.html)

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/130214) / small_tests / 86ce428720dc3275c00d0321c2bc7f4ec489fd50
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3710/130214/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3710/130214/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2022-07-19_07.36.32/lib.mongooseim.logs/run.2022-07-19_07.36.32/suite.log.html)

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/130298) / small_tests / 72d8994fd0a3056a90bdd71f31639a04edc4120a
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3710/130298/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3710/130298/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2022-07-19_10.55.14/lib.mongooseim.logs/run.2022-07-19_10.55.15/suite.log.html)

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/130348) / small_tests / f45ab32d9cd02ee2582e4f1d312e7a287d3e98eb
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3710/130348/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3710/130348/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2022-07-19_12.30.31/lib.mongooseim.logs/run.2022-07-19_12.30.32/suite.log.html)

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/130378) / small_tests / c4ac2e2244e6815b48b594d67826da867e3f4d17
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3710/130378/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3710/130378/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2022-07-19_13.14.48/lib.mongooseim.logs/run.2022-07-19_13.14.48/suite.log.html)

**Review by JanuszJakubiec — APPROVED** [review]
looks good

**Review by Premwoik — APPROVED** [review]
All good 👌🏻 500 lines less of doc definitions

File: big_tests/tests/graphql_domain_SUITE.erl [hunk 1]
```diff
@@ -43,271 +43,169 @@ domain_handler() ->
 
 init_per_suite(Config) ->
     case mongoose_helper:is_rdbms_enabled(?HOST_TYPE) of
-        true -> escalus:init_per_suite(init_admin_handler(Config));
-        false -> {skip, require_rdbms}
+        true ->
+            Config1 = ejabberd_node_utils:init(mim(), Config),
```
[the remark this answers is not in the document]
Comment (chrzaszcz) [reply big_tests/tests/graphql_domain_SUITE.erl:47]: It sets up the helpers for callin `mongooseimctl`

**Pull request merged**